libs/net/kafka/KafkaListener`.cs

```diff
@@ -141,6 +141,8 @@ public async Task ConsumeAsync(Func<ConsumeResult<TKey, TValue>, Task> action, C
             if (!this.IsReady) throw new InvalidOperationException("Consumer is not ready for consuming, it must be opened first.");
 
             this.IsConsuming = true;
+            _logger.LogDebug("Waiting to receive message from Kafka topics:'{topic}'", String.Join(", ", this.Consumer.Subscription));
```

Comment:
Helps to debug to know if the service is waiting for a message, otherwise it's not clear.